packages/bootleg-app-contracts/migrations/2_deploy_and_mint.js

```diff
@@ -1,17 +1,34 @@
 const BootlegToken = artifacts.require('BootlegToken');
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
 
-module.exports = async deployer => {
+module.exports = async (deployer, network, accounts) => {
+  let firstOwner = web3.utils.toHex(0);
+  const metaDataURI = 'https://ipfs.infura.io/ipfs/QmSomeHash';
+
+  // Bootleg Token ID
   const tokenId = web3.utils.toBN(1);
+  // should be the Band
+  firstOwner = accounts[0];
```

Comment:
nvm about the whole thing above, maybe actually remove the line above and...

```suggestion
  const firstOwner = process.env.BAND_ADDRESS || accounts[0];
```